wordpress/dto/WordpressTypesDTO.ts

```diff
@@ -1,11 +1,11 @@
 import {isString} from "../../modules/lodash";
 
-export enum WP_Types {
-    publish = "publish",
-    future = "future",
-    draft = "draft",
-    pending = "pending",
-    private = "private"
+export type WP_Types = {
```

Comment:
This probably should have a name like `WordpressType` with same name as the file name